client/src/CreateBitButton.jsx

```diff
@@ -26,21 +27,26 @@ class CreateBitButton extends Component {
   }
 
   render() {
-    const { filtering, createBit } = this.props;
-    return ([
-      <Button bsStyle="danger" onClick={this.show} key="create-bit-button">
-        Create new bit
-      </Button>,
-      <CreateBitModal show={this.state.show} createBit={createBit} onHide={this.hide} key="create-bit-modal"
-        allChars={filtering.get('chars')}
-        allStages={filtering.get('stages')}
-        allTags={filtering.get('standaloneTags')} />
-    ]);
+    const { filtering, profile, createBit } = this.props;
+    if (profile) {
+      return ([
+        <Button bsStyle="danger" onClick={this.show} key="create-bit-button">
+          Create new bit
+        </Button>,
```

Comment:
wtf i just noticed this comma-separated HTML tag nonsense

Reply:
Lol it's just a list of elements to render.